src/Workspaces/CoreTest/CodeCleanup/ReduceTokenTests.cs

```diff
@@ -2024,6 +2024,22 @@ End Module
             await VerifyAsync(code, expected);
         }
 
+        [Fact]
+        [WorkItem(14034, "https://github.com/dotnet/roslyn/issues/14034")]
+        [Trait(Traits.Feature, Traits.Features.ReduceTokens)]
+        public async Task ReduceIntegersWithDigitSeparators()
+        {
+            var source = @"
+Module Module1
+    Sub Main()
+        Dim x = 100_000
```

Comment:
This should have been just `0_0` to make it look like the emoji or whatever it's called these days.